go.mod

```diff
@@ -2,7 +2,10 @@ module sigs.k8s.io/cluster-api-provider-aws/v2
 
 go 1.19
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.3
+replace (
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.4
+	sigs.k8s.io/kind => sigs.k8s.io/kind v0.18.0
```

Comment:
Is it worth doing the same in hack/tools?

Reply:
hack/tools already have direct import of kind, so it would take affect, I have changed it to v0.18.0 already.